etl-pipeline/processes/record_clusterer.py

```diff
@@ -198,6 +271,7 @@ def _create_work_from_editions(self, editions: list, records: list[Record]):
         return record_manager.work, stale_work_ids
 
     def _index_works_in_elastic_search(self, work: Work):
+        """Indexes a single work in Elasticsearch."""
```

Comment:
Let's change the method name to _index_work_... that way it should be pretty clear!